src/gui/Window.qml

```diff
@@ -100,4 +113,51 @@ ApplicationWindow {
 
         Behavior on scale { HNumberAnimation { overshoot: 3; factor: 1.2 } }
     }
+
+    SystemTrayIcon {
+        property string iconPack: theme ? theme.icons.preferredPack : "thin"
+
+        visible: true
+        tooltip: qsTr("Mirage")
+        icon.source: `../icons/${iconPack}/tray-icon.png`
+
+        onActivated:
+            if (reason !== SystemTrayIcon.Context)
+                window.drawAttention()
+
+
+        menu: Menu {
+            MenuItem {
+                text: window.visible ? "Hide Mirage" : "Show Mirage"
```

Comment:
`"Open " + Qt.application.displayName` and `"Minimize to tray"` would be better for texts